packages/material/src/components/styles/abm-button.scss

```diff
@@ -14,7 +14,7 @@
   font-size: 14px;
   font-weight: 500;
   line-height: 36px;
-  color: rgb(var(--abm-primary-rgb, 33, 150, 243));
+  color: RGB(var(--abm-primary-rgb, 33, 150, 243));
```

Comment:
Please don't change the style of the CSS functions.

Reply:
Oh right. This is due to node-sass (if i'm not wrong) unable to compile rgb using css variables and default values. https://github.com/sass/node-sass/issues/2251

Reply:
The existing script works, and we shouldn't change valid CSS because of weird bugs in tools.